game/plugin/skills/woodcutting/meta.toml

```diff
@@ -0,0 +1,9 @@
+name = "woodcutting-skill"
```

Comment:
Now we have the `build.gradle` files we can delete the `meta.toml`s, so this can go